config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -281,7 +281,7 @@ snapshotter_version() {
         # Additional jobs could be created to cover version
         # skew, if desired.
         case "$kubernetes" in
-            1.20)
+            latest|release-1.20|1.20)
```

Comment:
is "master" also a possible value?

Reply:
Not in the current version of the script. There is one `snapshotter_version latest` invocation and one with `snapshotter $actual` where actual is set to `latest`.

But it can't hurt to include it, just in case.

I was also toying with the idea to use `1.[2-9][0-9]` and `release-1.[2-9][0-9]` as pattern to avoid having to update this when adding 1.21 and future version, but wasn't sure - might be overkill?

Reply:
Hmmm that's a good point. I think having a > check would be nice so we don't have to keep remembering to update this

Reply:
I've copied the `version_gt` from the hostpath deploy script and added "master".